azure-iot-device/azure/iot/device/iothub/inbox_manager.py

```diff
@@ -25,50 +25,16 @@ def __init__(self, inbox_type):
         :param inbox_type: An Inbox class that the manager will use to create Inboxes.
         """
         self._create_inbox = inbox_type
-        self.unified_message_inbox = self._create_inbox()
-        self.generic_method_request_inbox = self._create_inbox()
+        self.message_inbox = self._create_inbox()
+        self.method_request_inbox = self._create_inbox()
         self.twin_patch_inbox = self._create_inbox()
         self.client_event_inbox = self._create_inbox()
 
-        # These inboxes are used only for non-unified receives, using APIs which are now
-        # deprecated on the client. However we need to keep them functional for backwards
-        # compatibility
-        self.c2d_message_inbox = self._create_inbox()
-        self.input_message_inboxes = {}
-        self.named_method_request_inboxes = {}
-
-        # Set this value to True if want to only use unified message mode
-        self.use_unified_msg_mode = False
-
-    def get_unified_message_inbox(self):
+    def get_message_inbox(self):
```

Comment:
"Carter!", you say, "Why does this class need to exist anymore when all it does is provide single line methods??? Doesn't this contribute to the over-complexity of the client??"

Yes. Once the async pipeline is in place and we only have one type of inbox (Async ones), it's probably safe to completely remove the InboxManager as a class. But for now it still provides value in that it allows us to abstract out the sync vs async inboxes